docs/reference/flag-definitions.md

```diff
@@ -4,6 +4,12 @@ description: flagd flag definition
 
 # Flag Definitions
 
+## Metadata
```

Comment:
I think metadata should come go either right before or after "shared evaluators". I'd also recommend moving the example config down to that section. That way, we're keeping the first few examples as clean as possible before moving on to move advanced topics.

Reply:
Move to after "shared evaluators".